canvas_account_admin_tools/urls.py

```diff
@@ -10,7 +10,7 @@
     url(r'^not_authorized$', icommons_ui_views.not_authorized, name='not_authorized'),
     url(r'^tool_config$', views.tool_config, name='tool_config'),
     url(r'^lti_launch$', views.lti_launch, name='lti_launch'),
-    url(r'^icommons_rest_api/(?P<path>.*)$', views.icommons_rest_api_proxy, name='icommons_rest_api_proxy'),
     url(r'^account_dashboard$', views.dashboard_account, name='dashboard_account'),
     url(r'^course_info/', include('course_info.urls', namespace='course_info')),
+    url(r'^api/course/v2/', include('course_api.urls')),
```

Comment:
Fantastic.  And it looks like that's not something we can fix, unless we want to update icommons_rest_api and mandate that it's always included under a set namespace.

http://www.django-rest-framework.org/api-guide/routers/#usage
(sadly no link directly to it, just search for "namespace" in page)